src/rcc.rs

```diff
@@ -228,7 +266,9 @@ const HSI: u32 = 16_000_000; // Hz
 pub struct CFGR {
     hclk: Option<u32>,
     // should we use an option? it can really only be on/off
+    #[cfg(not(feature = "stm32l47x"))]
     hsi48: bool,
+    msi: MsiFreq,
```

Comment:
Could we have Option<MsiFreq> instead? That way we can remove the need for a NOMSI variant

Reply:
Done